api/src/opentrons/protocol_api/core/engine/deck_conflict.py

```diff
@@ -330,41 +345,50 @@ def _check_deck_conflict_for_8_channel(
     well_location_point = engine_state.geometry.get_well_position(
         labware_id=labware_id, well_name=well_name, well_location=well_location
     )
+    primary_nozzle = engine_state.pipettes.get_primary_nozzle(pipette_id)
 
     if not _is_within_pipette_extents(
         engine_state=engine_state, pipette_id=pipette_id, location=well_location_point
     ):
         # WARNING: (spp, 2023-11-30: this needs to be wired up to check for
         # 8-channel pipette extents on both OT2 & Flex!!)
         raise PartialTipMovementNotAllowedError(
-            "Requested motion with single H1 nozzle configuration"
-            " is outside of robot bounds for the 8-channel."
+            f"Requested motion with single {primary_nozzle} nozzle configuration"
+            f" is outside of robot bounds for the 8-channel."
         )
 
     labware_slot = engine_state.geometry.get_ancestor_slot_name(labware_id)
-    north_slot_number = get_north_slot(
-        _deck_slot_to_int(DeckSlotLocation(slotName=labware_slot))
-    )
-    if north_slot_number is None:
-        return
-
-    north_slot = DeckSlotName.from_primitive(
-        north_slot_number
-    ).to_equivalent_for_robot_type(engine_state.config.robot_type)
+    destination_slot = _deck_slot_to_int(DeckSlotLocation(slotName=labware_slot))
```

Comment:
Nit: I think this can be simplified to eliminate the conversion back and forth from `DeckSlotLocation`.

```suggestion
    destination_slot = labware_slot.as_int()
```

And then `_deck_slot_to_int()` can be deleted. It was helpful when the state view was returning a `DeckSlotLocation`, but it's unnecessary now that the state view is directly returning a `DeckSlotName`.